includes/wcpdf-functions.php

```diff
@@ -146,6 +146,7 @@ function wcpdf_pdf_headers( $filename, $mode = 'inline', $pdf = null ) {
 			header('Content-Type: application/pdf');
 			header('Content-Disposition: attachment; filename="'.$filename.'"'); 
 			header('Content-Transfer-Encoding: binary');
+			header("Content-Length: ".is_null($pdf) ? 0 : mb_strlen($pdf));
```

Comment:
WPCS: spaces

Reply:
@dwalkerpriv I see you have added some spaces in 0270f33, but WPCS specifies spaces between brackets too (not square brackets though). See: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage
You could apply the same rules to the rest of this file so that it's fully up to standard.

And for consistency it's good to use the same quotes for all `header()` calls (in this case single quotes)